Sources/antimony/JobEmitter.swift

```diff
@@ -76,6 +82,13 @@ public struct JobEmitter {
       target.isSwiftTarget ? target.flags.swift : []
     ].flatMap { $0 }
 
+    #if os(macOS)
+      arguments += [
+        "-sdk",
+        "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk"
```

Comment:
Can we dynamically detect this? Perhaps rely on the same mechanism as Windows (`$ENV{SDKROOT}`).

Reply:
BTW, longer term, I expect this to be removed (along with the triples). Those belong in the toolchain configuration, not embedded into the tool.